src/main/java/org/opensearch/ad/rest/RestGetAnomalyDetectorAction.java

```diff
@@ -58,17 +57,13 @@ public class RestGetAnomalyDetectorAction extends BaseExtensionRestHandler {
     private static final String GET_ANOMALY_DETECTOR_ACTION = "get_anomaly_detector";
     private static final Logger logger = LogManager.getLogger(RestGetAnomalyDetectorAction.class);
     private Settings settings;
-    private TransportService transportService;
     private SDKRestClient client;
-    private SDKClusterService clusterService;
     private ExtensionsRunner extensionsRunner;
 
     public RestGetAnomalyDetectorAction(ExtensionsRunner extensionsRunner, SDKRestClient client) {
         this.extensionsRunner = extensionsRunner;
```

Comment:
Since we're not using the extensionsRunner anywhere else besides the constructor to retrieve the settings, no need to keep it as a class field.

Reply:
@joshpalis doesn't have the write access. Resolving the comments on his behalf.